src/language-js/embed.js

```diff
@@ -578,12 +590,16 @@ function printHtmlTemplateLiteral(path, print, textToDoc, parser, options) {
 
   const contentDoc = mapDoc(
     stripTrailingHardline(
```

Comment:
Can we remove this `stripTrailingHardline` call?

Reply:
Found another one, and removed, end up we only use `stripTrailingHardline` for `embed`.